src/backend/OpenGL/impl.jl

```diff
@@ -19,6 +19,7 @@
 function ImGui_ImplOpenGL3_Init(glsl_version::Integer=130)
     io = GetIO()
     io.BackendRendererName = "imgui_impl_opengl3"
+    io.BackendFlags = io.BackendFlags | ImGuiBackendFlags_RendererHasVtxOffset;
```

Comment:
@inkydragon please delete this line OR change `glDrawElements` to `glDrawElementsBaseVertex`

Reply:
delete this line will cause `Assertion failed`